terraform/modules/ecs/ecs.tf

```diff
@@ -767,6 +798,23 @@ resource "aws_cloudwatch_event_rule" "send_job_alerts_daily_email_task" {
   schedule_expression = "${var.send_job_alerts_daily_email_schedule}"
 }
 
+resource "aws_ecs_task_definition" "send_expiry_alerts_task" {
+  family                   = "${var.ecs_service_web_task_name}_send_expiry_alerts_task"
+  container_definitions    = "${data.template_file.send_expiry_alerts_container_definition.rendered}"
+  requires_compatibilities = ["EC2"]
+  network_mode             = "bridge"
+  cpu                      = "256"
+  memory                   = "512"
+  execution_role_arn       = "${aws_iam_role.ecs_execution_role.arn}"
+  task_role_arn            = "${aws_iam_role.ecs_execution_role.arn}"
+}
+
+resource "aws_cloudwatch_event_rule" "send_expiry_alerts_task" {
+  name                = "${var.ecs_service_web_task_name}_send_expiry_alerts_task"
+  description         = "Send alerts to users who's alerts are due to expire"
+  schedule_expression = "${var.send_expiry_alerts_schedule}"
+}
```

Comment:
All this needs to be refactored using the new Terraform for declaring rake task definitions. Grab me if this isn't obvious.